packages/sdk/lib/base.ts

```diff
@@ -35,6 +35,11 @@ export type Event = {
   properties?: Properties;
   id?: string;
   version?: string;
+  metadata?: {
```

Comment:
Creating (and exporting?) an explicit Metadata type (just like we have an explicit Properties type) would be more consistent and (arguably) cleaner code.